eos-phone-home-if-up

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+[ "$IFACE" != "lo" ] || exit 0
```

Comment:
done, but not echoing something because it's already very clear from the journal whether this script does anything or not